e2e/cypress/integration/ui/compliance/large_compliance_report/large_compliance_report.spec.ts

```diff
@@ -12,9 +12,7 @@ describe('login the app', () => {
         username = admin.username;
         connector = admin.connector;
         cy.visit('/compliance/reports/nodes').then(() => {
-          cy.get('[data-cy=welcome-title]').should('exist').then(() => {
-              cy.get('[data-cy=close-x]').click();
-          });
+          cy.get('app-welcome-modal').invoke('hide');
```

Comment:
first, you need to check if the modal exists then it will close.
you have not added the code you are directly closing the modal.